lib/fastlane_core/itunes_connect/itunes_connect_helper.rb

```diff
@@ -61,8 +61,9 @@ def wait_for_preprocessing
         end
       end
 
-      def wait_for_elements(name)
+      def wait_for_elements(name, include_hidden = false)
         counter = 0
+        Capybara.ignore_hidden_elements = false if include_hidden
```

Comment:
You're right, we should backup the initial value and restore then.
Regarding move it to produce, it's ok for me. I did it here because I think is the best option, because who depends of `Capybara` is `fastlane_core`.
But you are the boss ;)
Let me know what you prefer.